pos_access_right/readme/DESCRIPTION.rst

```diff
@@ -5,3 +5,8 @@ This module extends Odoo Point Of Sale features, restricting possibility to cash
 * **PoS - Change Unit Price**: The cashier can change the unit price of a product in Point Of Sale;
 * **PoS - Many Orders**: The cashier can many orders at the same time;
 * **PoS - Delete Order**: The cashier can not delete a full order;
+* **PoS - Change Quantity**: The cashier can change the quantity of a product in Point Of Sale;
```

Comment:
It's mostly for system with orders syncing between  POSes, but can make a little sense in a single POS if you switch between Cashiers

Reply:
@yelizariev I think that for one POS it means that the current user of the system working with POS should not have access to change the current cashier and work on his behalf